roseau/load_flow/models/lines/line_characteristics.py

```diff
@@ -24,29 +26,29 @@ class LineCharacteristics:
     )
 
     @ureg.wraps(None, (None, None, "ohm/km", "S/km"), strict=False)
-    def __init__(self, type_name: str, z_line: np.ndarray, y_shunt: Optional[np.ndarray] = None):
+    def __init__(self, id: Id, z_line: np.ndarray, y_shunt: Optional[np.ndarray] = None) -> None:
```

Comment:
Maybe use a `star` too in the signature of this constructor to ensure keywords arguments

Reply:
It may be the same in the `TransformerCharacteristics`

Reply:
Oh yes, I did not realize.